docs/Contributing/API-for-contributors.md

```diff
@@ -671,12 +671,13 @@ This endpoint stores a profile to be assigned to a host at some point in the fut
 
 #### Parameters
 
-| Name                     | Type   | In   | Description                                                                                  |
-| ------------             | ------ | ---- | -----------------------------------------------------------                                  |
-| external_host_identifier | string | body | **Required**. The identifier of the host as generated by the external service (e.g. Puppet). |
-| host_uuid                | string | body | **Required**. The UUID of the host.                                                          |
-| profile                  | string | body | **Required**. The base64-encoded .mobileconfig content of the MDM profile.                   |
-| group                    | string | body | The group label associated with that profile. This information is used to generate team names if they need to be created. |
+| Name                     | Type    | In   | Description                                                                                  |
+| ------------             | ------- | ---- | -----------------------------------------------------------                                  |
+| external_host_identifier | string  | body | **Required**. The identifier of the host as generated by the external service (e.g. Puppet). |
+| host_uuid                | string  | body | **Required**. The UUID of the host.                                                          |
+| profile                  | string  | body | **Required**. The base64-encoded .mobileconfig content of the MDM profile.                   |
+| group                    | string  | body | The group label associated with that profile. This information is used to generate team names if they need to be created. |
+| exclude                  | boolean | body | If the profile shouldn't be delivered to this host. |
```

Comment:
For boolean properties, we've used the convention of starting descriptions with "Whether [...]". I realize that's a little tricky here since `true` means _don't_ do something...

Would it work to say something like:
```suggestion
| exclude                  | boolean | body | Whether to skip delivering the profile to this host. |
```

Reply:
much better, thanks!